tests/net/tcp/src/main.c

```diff
@@ -96,6 +96,7 @@ static struct sockaddr_in6 peer_addr_v6_s = {
 static struct net_if *iface;
```

Comment:
Please rename the test commit to
`tests: net: tcp: ....`
so that it is easier to search from commit log.

Reply:
Done